debian/control

```diff
@@ -159,6 +159,16 @@ Description: nymea.io plugin for M-TEC heat pumps
  This package will install the nymea.io plugin for M-TEC heat pumps
 
 
+Package: nymea-plugin-stiebeleltron
+Architecture: any
+Section: libs
+Depends: ${shlibs:Depends},
+         ${misc:Depends},
+         nymea-plugins-modbus-translations
```

Comment:
We can remove this dependency if we are installing the translation file (see `nymea-plugin-stiebeleltron.install.in` comment)

Reply:
See e852d32